test/groovy/WriteTemporaryCredentialsTest.groovy

```diff
@@ -111,11 +132,12 @@ class WriteTemporaryCredentialsTest extends BasePiperTest {
     @Test
     void credentialsFileWrittenAndRemoved() {
         def credential = [alias: 'ERP', credentialId: 'erp-credentials']
-        fileExistsRule.registerExistingFile('systems.yml')
+        fileExistsRule.registerExistingFile('./systems.yml')
+        fileExistsRule.registerExistingFile('./credentials.json')
 
         nullScript.commonPipelineEnvironment.configuration = [stages: [myStage:[
             credentials: [credential],
-            credentialsDirectory: './',
+            credentialsDirectories: ['./', 'integration-test/'],
```

Comment:
integrations-test**s**?

Reply:
The provided path is not important as there is no systems.yml in `integration-test/` and no assertion using it. But maybe I could add an assertion that checks for the size of writeFileRule.files since I should also make sure the file is only written to `./`